make/go/dep_protoc_gen_go.mk

```diff
@@ -7,9 +7,8 @@ $(call _assert_var,CACHE_VERSIONS)
 $(call _assert_var,CACHE_BIN)
 
 # Settable
-# https://github.com/protocolbuffers/protobuf-go/releases 20240611 checked 20240612
-# TODO: Change back to released version once bug is fixed
-PROTOC_GEN_GO_VERSION ?= 94ecbc26168965a670a0f7cf86f658131c790a9c
+# https://github.com/protocolbuffers/protobuf-go/releases 20240906 checked 20240916
+PROTOC_GEN_GO_VERSION ?= v1.34.3-0.20240906163944-03df6c145d96
```

Comment:
Why is this being done? We have always used commits here, not Go pseudo-versions

Reply:
Looks like this was approved in https://github.com/bufbuild/makego/pull/153/files

Reply:
OK it's fine